todo_app/templates/index.html

```diff
@@ -22,103 +22,79 @@ <h1 class="display-4">To-Do App</h1>
 		<table>
 			<h3><b>To do</b></h3>
 
-			{%for item in view_model.todolist : %}
+			{%for x in my_items : %}
 			<div>
 				<br>
-
-				<tr>{{item.name}}</tr>
+				<tr> {{x.name }}</tr>
 				<tr>
 
-					<p><b>Move from doing to done</b></p>
-					<form action="/update" method="post">
-						<input type="hidden" name="id" value="{{item.id}}">
-						<input type="submit" value="Update to done!!!">
-					</form>
-			</div>
-		</tr>
-			{% endfor %}
-			<br>
-			<p>Add To do Task
-			</p>
-			<form action="/create" method="POST">
-				<input type="text" name="title" id="title" value="title" required>
-				<input type="submit" id="create" value="Create!">
-			</form>
-		</table>
+				</tr>
+				<p><b>Move from doing to done</b></p>
+				<form action="/update" method="post">
+					<input type="hidden" name="id" value="{{x.id}}">
+					<input type="submit" value="Update to done!">
+				</form>
+				{% endfor %}
+				<br>
+				{% if user =="writer" %}
```

Comment:
Good, but what about all of the other forms?

Reply:
I have added an else statement, thank you